### PR TITLE
Fixed volume updating

### PR DESCRIPTION
**Pull request by BernardZhao**
The last issue with #7. Volume should work perfectly now, given that the mpd conf continues to be stable.

### CODE REVIEW

File: priv/static/site.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ var main_socket_handler = function (event) {
     var span_currentvol = document.getElementById("currentvol");
 		removeChildren(span_currentvol);
 		span_currentvol.appendChild(document.createTextNode(serverstate["volume"]));
+    document.getElementById("volslider").value = serverstate["volume"];
```
Comment (PotatoParser) [inline priv/static/site.js]: Is there a missing indent here?

**Review by PotatoParser — CHANGES_REQUESTED** [review]

File: priv/static/site.js (hunk 1 above)
Comment (BernardZhao) [reply priv/static/site.js]: Way ahead of you

**Comment by BernardZhao** [comment]
@encadyma and I thought it would just be easier to format this entire file per Prettier convention. Rip tabs.

File: priv/static/site.js (hunk 1 above)
Comment (PotatoParser) [reply priv/static/site.js]: Bless

**Review by encadyma — APPROVED** [review]
Looks good for now. As for the future of site.js, a rewrite is definitely in order..

**Review by PotatoParser — APPROVED** [review]
LGTM!

**Pull request merged**